includes/api/ApiViewDumps.php

```diff
@@ -54,21 +54,21 @@ public function execute() {
 
 				$buildResults[] = [
 					'filename' => $dump->dumps_filename,
-					'link' => $this->getDownloadUrl( $config, $dump ),
+					'link' => $this->getUrl( $config, $dump ),
 					'time' => $dump->dumps_timestamp ?: '',
 					'type' => $dump->dumps_type,
 				];
 			}
 		}
 		$this->getResult()->addValue( null, $this->getModuleName(), $buildResults );
 	}
-
-	private function getDownloadUrl( $config, $dump ) {
+	
+	private function getUrl( $config, $dump ) {
```

Comment:
Use `getDownloadUrl()` again, since only one function now.

```suggestion
	private function getDownloadUrl( $config, $dump ) {
```